### PR TITLE
Replace LOG(FATAL) with LOG(DFATAL)

### LINKED ISSUE #5068: Reduce the usages of LOG(FATAL) or other self-descruction measures as many as possible.
RT.

To improve the stability and maximize the running hours of services as much as possible.

Reasons to self-destruct:

1. Data may be corrupted or lost, if the service continues to run.
2. (Add more reasons here, if any)

### PR DESCRIPTION
**Pull request by veezhang**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 

fix #5068

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/kvstore/raftex/RaftPart.cpp [hunk 1]
```diff
@@ -1091,7 +1091,11 @@ void RaftPart::processAppendLogResponses(const AppendLogResponses& resps,
               [self = shared_from_this(), term = term_] { self->onLeaderReady(term); });
         }
       } else {
-        LOG(FATAL) << idStr_ << "Failed to commit logs";
+        LOG(DFATAL) << idStr_ << "Failed to commit logs";
+        checkAppendLogResult(code);
+        VLOG(3) << idStr_ << "commitLogs failed";
+        iter.commit(code);
```
Comment (xtcyclist) [inline src/kvstore/raftex/RaftPart.cpp:1097]: Why add these three lines here? Why not simply return?

Comment (veezhang) [reply src/kvstore/raftex/RaftPart.cpp:1097]: Like here https://github.com/vesoft-inc/nebula/pull/5078/files/ddbb5a96236c6753494f533c74044b1dc8d33960#diff-f56618b12fe75ed2289df1551818fdda84ea75d2f5c8b4c406fc196e06e7768bL1062-L1067 

I think it's necessary to commit the error code too.

Comment (veezhang) [reply src/kvstore/raftex/RaftPart.cpp:1097]: @xtcyclist Of course, you are more familiar with it. If you think it is unnecessary, I can change it to return directly.

Comment (xtcyclist) [reply src/kvstore/raftex/RaftPart.cpp:1097]: Better remove these three lines. I think we'd better not adding new codes here.

**Review by xtcyclist — DISMISSED** [review]

**Review by critical27 — COMMENTED** [review]
Maybe you can revert the changes in dir `src/kvstore`. They rarely fail, and if it really happens, we'd better crash there.

**Comment by veezhang** [comment]
> Maybe you can revert the changes in dir `src/kvstore`. They rarely fail, and if it really happens, we'd better crash there.

ok

**Review by critical27 — APPROVED** [review]

**Review by xtcyclist — APPROVED** [review]

**Pull request merged**